docs/test_runner.py

```diff
@@ -4,9 +4,9 @@
 import pytest
 
 # Collecting all the Python files in the scripts directory
-script_files = [f for f in os.listdir("test_scripts/") if f.endswith('.py')]
+script_files = [f for f in os.listdir("test_scripts/") if f.endswith(".py")]
```

Comment:
Just checking: Is this from the formatter? What version of black do you have installed? I'm assuming this was done via pre-commit?

Reply:
Yes, this is from the formatter. I'm using `22.10.0`